styles/email/_email-variables.scss

```diff
@@ -0,0 +1,19 @@
+// /**
+//  * Copyright © 2013-2017 Magento, Inc. All rights reserved.
+//  * See COPYING.txt for license details.
+//  */
+
+//
+//  Email variable overrides
+//  _____________________________________________
+
+//  This file should be used to set theme-specific variables for emails.
+//  By default, emails will inherit variables from the Magento UI Library and the _theme.less and _variables.less files
+//  in a custom theme. Use this file if you want to set specific variables for emails.
+//  For example, you can make the email font, typography, colors, etc unique from the frontend theme.
+
+//  Example usage:
+//  @link__color: @color-dark-green1;
+//  @link__text-decoration: underline;
+//
+//  @email__background-color: @color-heathered-grey; // Change background to darker grey color
```

Comment:
Should this file be empty?

Reply:
Some examples. User can define / change variables for email styles as in styles/email.scss and styles/email-inline.scss we have

```
@import 'variables'; // Local theme variables
@import 'email/email-variables'; // Theme variables for emails
...
@import 'email/email-base-inline';
@import 'email/email-extend-inline';
```